scripts/publish_external.sh

```diff
@@ -1,11 +1,8 @@
 # undo permissions change on scripts/publish_external.sh
 git checkout -- scripts
 
-# Use newer, faster npm
-npm install -g npm@5.0.3
-
 # Install required dependencies
-npm install -g lerna
+yarn global add lerna
```

Comment:
Is there a reason to add this globally? It's already installed as a devDependency in the root package.

Also, is the following line to bootstrap no longer necessary?